src/AppBundle/Resources/views/Report/ProfDeputyCostsEstimate/managementCosts.html.twig

```diff
@@ -0,0 +1,41 @@
+{% extends 'AppBundle:Layouts:application.html.twig' %}
+{% set transOptions = {'%client%': report.client.firstname | e } %}
+
+{% import 'AppBundle:Macros:macros.html.twig' as macros %}
+
+{% set translationDomain = "report-prof-deputy-costs-estimate" %}
+{% trans_default_domain translationDomain %}
+{% set page = 'managementCosts' %}
+
+{% block htmlTitle %}{{ (page ~ '.htmlTitle') | trans }}{% endblock %}
+{% block pageTitle %}{{ (page ~ '.pageTitle') | trans }}{% endblock %}
+{% block pageTitleClass %}heading-large{% endblock %}
+
+{% block breadcrumbs %}{{ macros.breadcrumbs(report) }}{% endblock %}
+
+{% block linkBack %}
+    {{ macros.linkBackStep(backLink, 'back' | trans({}, 'common')) }}
+{% endblock %}
+
+{% block pageContent %}
+
+    <p class="heading-small push-half--bottom  text">
+        {{ (page ~ '.pageSectionDescription') | trans(transOptions) }}
+    </p>
+
+    {{ form_start(form, {attr: {novalidate: 'novalidate'}}) }}
+
+    <div class="{{ profDeputyManagementCost.moreDetails is defined ? ' js-details-expander ' : '' }}">
```

Comment:
ticket doesnt mention having more details - hence why a lot of this is not needed :)